src/Entity/InformationProduct.php

```diff
@@ -119,6 +119,17 @@ class InformationProduct extends Entity
      */
     private $file;
 
+    /**
+     * The collection of types for this Information Product.
+     *
+     * @var Collection
+     *
+     * @Serializer\MaxDepth(1)
+     *
+     * @ORM\OneToMany(targetEntity="InformationProductTypeDescriptor", mappedBy="InformationProduct")
```

Comment:
Since there is not reverse, the mappedBy may not be needed. (As it doesn't exist)

Reply:
It requires that mappedBy or else it breaks